scalalib/src/mill/scalalib/Versions.scala

```diff
@@ -1,8 +1,8 @@
 package mill.scalalib
 
+// Keep this synchronized with buildVersions.sc
 object Versions {
```

Comment:
Rather than duplicating this from the `.sc` file, could we inject these from the Mill build into the Mill executable as environment variables? We already do that in a number of places, e.g. `-DMILL_VERSION `, so this would fit right in